packages/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "react-instantsearch-dom-maps",
```

Comment:
this is because the version.js script in React InstantSearch dom maps having the wrong path probably